File: src/Core/Houston.Application/ViewModel/ConnectorFunctionViewModels/ConnectorFunctionSummaryViewModel.cs
namespace Houston.Application.ViewModel.ConnectorFunctionViewModels {
	public class ConnectorFunctionSummaryViewModel {
		public Guid Id { get; set; }

		public string Name { get; set; } = null!;

		public string FriendlyName { get; set; } = null!;

		public string Connector { get; set; } = null!;

		public string Version { get; set; } = null!;

		public BuildStatus BuildStatus { get; set; }

		public string CreatedBy { get; set; } = null!;

		public DateTime CreationDate { get; set; }

		public string UpdatedBy { get; set; } = null!;

		public DateTime LastUpdate { get; set; }
	}
}

File: src/Web/WebAngular/src/app/domain/view-models/connector-function-summary.view-model.ts
import { BuildStatus } from "../enums/build-status.enum";

export interface ConnectorFunctionSummaryViewModel {
	id: string;
	name: string;
	friendlyName: string;
	connector: string
	version: string;
	buildStatus: BuildStatus;
	createdBy: string
	creationDate: string;
	updatedBy: string;
	lastUpdate: string;
}
